README.md

```diff
@@ -269,6 +272,29 @@ properties of the model.
 [stac-raster-stats]: https://github.com/stac-extensions/raster?tab=readme-ov-file#statistics-object
 [stac-band-names]: https://github.com/stac-extensions/eo?tab=readme-ov-file#common-band-names
 
+#### Model Band Object
+
+| Field Name | Type   | Description                                                                                                                 |
+|------------|--------|-----------------------------------------------------------------------------------------------------------------------------|
+| name       | string | **REQUIRED** Name of the band referring to an extended band definition (see [Bands](#bands-and-statistics).                 |
+| format     | string | The type of expression that is specified in the `expression` property.                                                      |
+| expression | \*     | An expression compliant with the `format` specified. The expression can be any data type and depends on the `format` given. |
```

Comment:
Was initially a bit confused reading this. Should it be rephrased to 

```suggestion
| expression | \*     | An expression compliant with the `format` specified. The expression can be applied to any numerical data type and depends on the `format` given. |
```

Reply:
Yes for the "applied to". 
I think we should avoid the "numerical" restriction. Not sure if there would realistically be any non-numeric cases, but I prefer to leave the flexibility.

Reply:
ok sounds good, agreed